### PR TITLE
[aws-c-http,aws-c-auth,aws-c-s3,aws-c-mqtt,aws-c-common,aws-sdk-cpp,aws-crt-cpp] update to 0.7.14, 0.7.6, 0.3.24, 0.9.9, 0.9.9,1.11.201,0.24.7

### PR DESCRIPTION
**Pull request by jiayuehua**
- [X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [X] SHA512s are updated for each updated download
- [ ] ~The "supports" clause reflects platforms that may be fixed by this new version~
- [ ] ~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~
- [ ] ~Any patches that are no longer applied are deleted from the port's directory.~
- [X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [X] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/aws-c-common/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-c-common",
-  "version": "0.9.4",
+  "version": "0.9.8",
```
Comment (jimwang118) [inline ports/aws-c-common/vcpkg.json:3]: The latest upstream version is 0.9.9.

File: ports/aws-c-s3/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-c-s3",
-  "version": "0.3.17",
+  "version": "0.3.23",
```
Comment (jimwang118) [inline ports/aws-c-s3/vcpkg.json:3]: The latest upstream version is 0.3.24.

File: ports/aws-crt-cpp/vcpkg.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-crt-cpp",
-  "version": "0.23.1",
+  "version": "0.24.5",
```
Comment (jimwang118) [inline ports/aws-crt-cpp/vcpkg.json:3]: The latest upstream version is 0.24.7.

File: ports/aws-sdk-cpp/vcpkg.json [hunk 4]
```diff
@@ -1,8 +1,7 @@
 {
   "$note": "Automatically generated by generateFeatures.ps1",
   "name": "aws-sdk-cpp",
-  "version": "1.11.169",
-  "port-version": 2,
+  "version": "1.11.200",
```
Comment (jimwang118) [inline ports/aws-sdk-cpp/vcpkg.json:4]: The latest upstream version is 1.11.201.

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

**Review by jimwang118 — APPROVED** [review]

**Comment by autoantwort** [comment]
This resulted in https://github.com/microsoft/vcpkg/issues/35182

**Pull request merged**